### DISCUSSION TITLE
Update shipping method no permissions

### DISCUSSION THREAD

**Discussion by PatrickVerberkmoes** [swellstores/community#454] [Help] [answered]
Hello,

Ik try to update my shipping method when i add a item to the cart , but when i do that i get this error:

```javascript
error: {code: "permission_error", message: "You are not allowed to update `shipping`", param: "shipping"}
code: "permission_error"
message: "You are not allowed to update `shipping`"
param: "shipping"
``` 

How can i fix this error?

**Comment by sags95** [comment, marked as answer]
@PatrickVerberkmoes Can you provide the code you're using to update the service?

**Reply by PatrickVerberkmoes** [reply]
```
  await useSwellApi<SwellCart>('/cart/update', {
                method: 'POST',
                body: {
                    shipping: {
                        method: 'standard',
                    },
                },
            });
```

**Reply by logeshswell** [reply]
Can you provide swell-js code snippet to assist you better?
Otherwise, it would also be very helpful if you could share the curl request with request payload and response body that throws this error.
